src/triggers/DeletionRules.trigger

```diff
@@ -0,0 +1,18 @@
+trigger DeletionRules on Expense__c (after update, after delete) {
```

Comment:
It's recommended to use the object's name as part of a trigger's name. For example, this trigger could be called ExpenseDeletionRules